### PR TITLE
add ability to use a custom formatter for values

### PR DESCRIPTION
**Pull request by blackmad**
Hi,

I added some functionality to be able to pass in a custom formatter for values in this library. It means that you can, for example, linkify http links in your json.

I wasn't sure how to generate gh-pages for this project, so I didn't update that branch.

Let me know if you like this

### CODE REVIEW

File: src/components/simple-text.vue [hunk 1]
```diff
@@ -30,11 +29,22 @@
...
+      defaultFormatter (data) {
         let text = data + ''
         if (this.dataType === 'string') text = `"${text}"`
         if (this.showComma) text += ','
         return text
+      },
+
+      textFormatter (data) {
+        if (this.customValueFormatter) {
+          console.log(this.customValueFormatter)
```
Comment (leezng) [inline src/components/simple-text.vue]: There are several log prints that must be removed.

File: example/App.vue [hunk 2]
```diff
@@ -228,6 +235,16 @@ export default {
     },
     handleChange (newVal, oldVal) {
       console.log('newVal: ', newVal, ' oldVal: ', oldVal)
+    },
+    customLinkFormatter(data) {
+      console.log(data)
```
Comment (leezng) [inline example/App.vue]: here

**Comment by blackmad** [comment]
deleted! sorry about that, should have checked my code more carefully

**Comment by leezng** [comment]
The gh-pages will be generated automatically when merge into master.

**Pull request merged**